components/NavigationRail.tsx

```
@@ -0,0 +1,142 @@
+    : mailboxes?.[0];
+
+  useEffect(() => {
+    if (!mailboxSlug && mailboxes && mailboxes.length > 0) {
+      setMailboxSlug(mailboxes[0]?.slug || "");
+    }
+  }, [mailboxes, mailboxSlug]);
+
+  useEffect(() => {
+    // Try to extract mailbox slug from the path
+    const match = pathname.match(/\/mailboxes\/([^/]+)/);
+    if (match && match[1]) setMailboxSlug(match[1]);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)